### ISSUE TITLE
Update `{report}` and `{modelbased}` on CRAN

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/easystats#345]
It has been over 20 weeks since the last release.

<img width="367" alt="Screenshot 2023-01-12 at 22 11 36" src="https://user-images.githubusercontent.com/11330453/212182107-d3af1869-0771-4509-9965-fc28995f0865.png">

**Comment by strengejacke** [comment]
See https://github.com/easystats/modelbased/issues/221

**Comment by rempsyc** [comment]
Currently, `report` has a remote dependency on the development version of `effectsize`. Does that mean `effectsize` needs to be submitted to CRAN first?

**Comment by IndrajeetPatil** [comment]
Yes, that's true! In that case, we can wait for a couple of weeks, update effectsize and then report.

**Comment by IndrajeetPatil** [comment]
@mattansb Do you have an estimate for when you will be updating _effectsize_ on CRAN? 

_report_ should be updated soon, but that's only possible after _effectsize_ is updated.

**Comment by mattansb** [comment]
I wasn't planning on updating until I was done with https://github.com/easystats/effectsize/pull/552, but there's no hurry - should I push a minor update now?

**Comment by IndrajeetPatil** [comment]
Okay, then we can wait for a couple of weeks for https://github.com/easystats/effectsize/pull/552 to be finished. 
If not, you can push a minor release. Thanks!

**Comment by mattansb** [comment]
I'm not sure when I'll have time to finish https://github.com/easystats/effectsize/pull/552, so I'll go a head and push a minor release (https://github.com/easystats/effectsize/pull/558).

**Comment by IndrajeetPatil** [comment]
@rempsyc _effectsize_ update is on CRAN. So you can submit _report_ in a couple of days.

**Comment by rempsyc** [comment]
Yeah! I can also resubmit `rempsyc` as I was waiting on effectsize since we added the 95% CI :-)

**Comment by mattansb** [comment]
> I was waiting on effectsize since we added the 95% CI :-)

Of what?

**Comment by IndrajeetPatil** [comment]
I think he means that he was waiting for _effectsize_ update before he could update [his own package](https://cran.r-project.org/web/packages/rempsyc/index.html) on CRAN.

**Comment by rempsyc** [comment]
In `rempsyc` I depend on `effectsize` 0.8.3 for the `sr2` that we added this version since I report it by default in all my lm functions.

**Comment by IndrajeetPatil** [comment]
@rempsyc Has `{report}` been submitted to CRAN?

**Comment by rempsyc** [comment]
Tomorrow. Trying to keep actual coding to-dos for weekends for now otherwise I'll never graduate 😬

**Comment by rempsyc** [comment]
@IndrajeetPatil I'm ready to submit report to CRAN as soon as easystats/report#335 is merged. There is a note because we're changing the maintainer. I've read that the former maintainer needs to email CRAN to give approval, but we'll see I guess.

**Comment by rempsyc** [comment]
`report`: Submitted

**Comment by rempsyc** [comment]
> Dear maintainer,
>  
> package report_0.5.6.tar.gz has been auto-processed.
> We are waiting for confirmation from the old maintainer address now.

**Comment by rempsyc** [comment]
> Thanks, on its way to CRAN.

**Comment by IndrajeetPatil** [comment]
@rempsyc Great! Can you please also create a new GitHub release?

**Comment by rempsyc** [comment]
Thanks for the reminder. The GH release is done now.

**Comment by IndrajeetPatil** [comment]
@DominiqueMakowski Only _bayestestR_ update is pending now.

**Comment by IndrajeetPatil** [comment]
Closing in favour of #355
